KosherCocoa/Library/Headers/KCJewishHolidays.h

```diff
@@ -84,7 +84,9 @@ enum kYomimTovim {
     kYomHashoah,
     kYomHazikaron,
     kYomHaatzmaut,
-    kYomYerushalayim
+    kYomYerushalayim,
+    kLagBaomer,
+    kShushanPurimKatan
 };
```

Comment:
Added these entries, but unsure about where the translations go.

Reply:
Good question, they're in [`/KosherCocoa/Library/Core/Calendar/Chagim/KCJewishHoliday.m`][2].

[2]: https://github.com/MosheBerman/KosherCocoa/blob/2cb3bac78d66c09ab09d762b074c198ccb8efb45/KosherCocoa/Library/Core/Calendar/Chagim/KCJewishHoliday.m#L84